### PR TITLE
added simple examples for os and k8s

### PR DESCRIPTION
**Pull request by jdob**
If these land, I'll update the docs README to mention that these exist. If we don't land these for whatever reason, it doesn't make sense to mention there since there is only a single, very basic example in here already.

### CODE REVIEW

File: examples/iso/k3s-single-node.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+apiVersion: 1.0
+image:
+  arch: x86_64
+  imageType: iso
+  baseImage: SLE-Micro.x86_64-5.5.0-Default-SelfInstall-GM2.install.iso
+  outputImageName: single-node-k3s-no-selinux.iso
+kubernetes:
+  version: v1.28.7+k3s1
```
Comment (atanasdinov) [inline examples/iso/k3s-single-node.yaml:8]: The supported version is 1.28.8 in Edge 3.0.0:
```suggestion
  version: v1.28.8+k3s1
```

File: examples/iso/k3s-single-node.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  network:
+    apiVIP: 192.168.100.19
+  nodes:
+    - hostname: node1.suse.com
+      type: server
```
Comment (atanasdinov) [inline examples/iso/k3s-single-node.yaml:13]: I'd drop this. It's ignored for single nodes.

File: examples/raw/k3s-single-node.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+apiVersion: 1.0
+image:
+  arch: x86_64
+  imageType: raw
+  baseImage: SLE-Micro.x86_64-5.5.0-Default-GM.raw
+  outputImageName: single-node-k3s-no-selinux.raw
+kubernetes:
+  version: v1.28.7+k3s1
```
Comment (atanasdinov) [inline examples/raw/k3s-single-node.yaml:8]:
```suggestion
  version: v1.28.8+k3s1
```

File: examples/raw/k3s-single-node.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  network:
+    apiVIP: 192.168.100.19
+  nodes:
+    - hostname: node1.suse.com
+      type: server
```
Comment (atanasdinov) [inline examples/raw/k3s-single-node.yaml:13]: Same as above.

**Review by atanasdinov — APPROVED** [review]

File: examples/iso/k3s-single-node.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  manifests:
+    urls:
+      - https://raw.githubusercontent.com/dbw7/misc/main/k8s-examples/apache-example.yaml
```
Comment (atanasdinov) [inline examples/iso/k3s-single-node.yaml:16]: Maybe a more official manifest is better here? Afraid that Danny's repository might turn private at some point and invalidate this. :D

Comment (e-minguez) [reply examples/iso/k3s-single-node.yaml:16]: https://k8s.io/examples/application/nginx-app.yaml ?

File: examples/iso/k3s-single-node.yaml (hunk 2 above)
Comment (jdob) [reply examples/iso/k3s-single-node.yaml:13]: This meaning the type attribute right, not the entire node reference?

File: examples/iso/k3s-single-node.yaml (hunk 5 above)
Comment (jdob) [reply examples/iso/k3s-single-node.yaml:16]: Lol, shit, I didn't even look that closely, I just copied it from our tests. Will fix.

File: examples/iso/k3s-single-node.yaml (hunk 2 above)
Comment (atanasdinov) [reply examples/iso/k3s-single-node.yaml:13]: No - the entire `nodes` section.

**Review by atanasdinov — APPROVED** [review]

**Review by fdegir — APPROVED** [review]
works on my machine

**Pull request merged**